### PR TITLE
Update pull request template; private names >2y

### PR DESCRIPTION
**Pull request by dnsguru**
Update to ensure the rationale includes an attestation of >2 years registration term and a commitment to maintaining at least >1 year of registration term per #1109

* [x] Description of Organization
* [ ] Reason for PSL Inclusion
* [ ] DNS verification via dig
* [ ] Run Syntax Checker (make test)

Description of Organization
====

Organization Website: 

Reason for PSL Inclusion
====

DNS Verification via dig
=======

make test
=========

### CODE REVIEW

**Comment by dnsguru** [comment]
@sleevi I tread lightly on this - didn't want to just merge this in to satisfy #1109

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -12,6 +12,12 @@ see https://github.com/publicsuffix/list/pull/615
 * [ ] DNS verification via dig
 * [ ] Run Syntax Checker (make test)
 
+If submission includes entries in the PRIVATE section:
+I attest that:
+* [ ] Domain(s) submitted have 2 or more years before they expire.
+* [ ] I (or my organization) shall maintain the registrations in good standing and will keep the expiry date >1 year at all times.
+* [ ] I understand that my entry could be revoked or rejected where these conditions are not met.
+
```
Comment (sleevi) [inline .github/pull_request_template.md:20]: Eh, I'm sort of on the fence about the whole "I attest" approach. That feels very formal and Serious and Heavy Weight and there's not much behind it.

Perhaps just:
```suggestion
* [ ] Each domain listed in the PRIVATE section has at least two years remaining on registration.
```

WDYT?

**Comment by dnsguru** [comment]
I am good with your change, with a tweak.

How about:    "..has *and shall maintain* at least.."

My rationale on keeping that ongoing commitment makes it possible for some
later automation to review, flag, comment or remove entries with domains
that have expired or are close to it.

**Review by sleevi — APPROVED** [review]

**Comment by dnsguru** [comment]
Squashmerged.  thanks.  @sleevi  @weppos - PRIVATE section entries' PR will now require this extra box ticked after 1115

**Comment by dnsguru** [comment]
Thank you @sleevi - can you take a quick peek at and approve #1116 ?

I was sweeping through issues and saw an opportunity to address #1027 while
modifying this, and I'd already SquashMerged #1113

**Pull request merged**